src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    if (String(a) < String(b)) {
+      return -1;
+    }
+
+    if (String(a) > String(b)) {
+      return 1;
+    }
+
+    return 0;
```

Comment:
You can move the default compare function to a separate function and pass its name to the default parameter value